Echo.py

```diff
@@ -3,17 +3,12 @@
 # Programmed by J. Robert B.
 
 import time
-
-def is_number(x):
-    return True
     
 def enput(prompt):
-    #capture raw input
-    #decide if number
-    
-    #if number return it
-    # else exit program and error?
-    return input(prompt)
+    try:
+        return input(prompt)
+    except NameError:
+        print "\"", input(prompt), "\"", " is not even a number, you fool."
```

Comment:
At first glance I would think this would cause you problems because you don't know if `input(prompt)` will be safe?